serverless/messaging/reference-content/migration.mdx

```diff
@@ -0,0 +1,173 @@
+
+To migrate, you must **recreate your resources** in the new version. Remember that the new version of the API has no namespaces: you will simply **activate the SNS protocol** and then directly create your credentials. SQS and SNS have been separated, so if you want to use both you will need to activate both separately and create separate credentials for both. Full documentation will be available. 
+
+<Message type="tip">
+
+If you previously had multiple namespaces to separate your SNS resources, we recommend that you use [Scaleway Projects](/console/my-project/concepts/#project) with the new version to handle this. By creating multiple Projects, activating SNS in each one, and then creating SNS credentials in each Project, you will be able to achieve separation of resources, and credentials scoped to the SNS broker of each Project.
+
+</Message>
+
+You can recreate your credentials either via the [console](https://console.scaleway.com/) (once the new version is in production) or via the new `v1beta1` API directly. In terms of the API, the [current `v1alpha1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) version will continue to be available 30 days after migration/General Availability, co-existing with the new `v1beta1` version. The documentation for both will be accessible via the [Scaleway Developers website](https://www.scaleway.com/en/developers/api) via a dropdown at the top of the Messaging and Queuing API page. 
+
+Note the following important information about the migration/crossover period, when both versions of the API will exist:
+
+**Before release of new version**
+- Only the `v1alpha1` API is available. All your resources exist in this API.
+- The console is based on this API, so when you view your resources in the console, you are viewing resources in the `v1alpha1` API.
+
+**At the time that SQS and NATS go into GA**
+- The `v1alpha1` API is still available. Your resources continue to exist in this API until you delete them.
+- The `v1beta1` API also becomes available. You should recreate your resources in this API.
+- The console now uses the `v1beta1` API. **You will no longer be able to see or manage your old resources from the `v1alpha1` API via the console**. Any resources now created via the console exist in the `v1beta1` API.
+
+**30 days afterwards**
+- The `v1alpha1` API is decommissioned. Any resources still existing on this API are deleted.
+
+**The URL/endpoints for your Messaging and Queuing SNS service and for your topics will not be the same in the two versions**. Make sure that when you recreate your resources with the new version, you remember to update the endpoint in any code, scripts, applications and architectures that were using  the old endpoint. If you do not do so, they will continue to access the "old" resources rather than the new ones you have created for the new version.
```

Comment:
```suggestion
**The URL/endpoints for your Messaging and Queuing SNS service and for your topics will not be the same in the two versions**. Make sure that when you recreate your resources with the new version, you remember to update the endpoint in any code, scripts, applications and architectures that were using the old endpoint. If you do not do so, they will continue to access the "old" resources rather than the new ones you have created for the new version.
```